openedx/adg/lms/webinars/helpers.py

```diff
@@ -30,7 +36,8 @@ def send_webinar_emails(template_slug, webinar, recipient_emails, send_at=None):
         'webinar_id': webinar.id,
         'webinar_title': webinar.title,
         'webinar_description': webinar.description,
-        'webinar_start_time': webinar.start_time.strftime("%B %d, %Y %I:%M %p %Z")
+        'webinar_start_time': webinar.start_time.astimezone(pytz.timezone(WEBINAR_DEFAULT_TIME_ZONE)).strftime(
+            WEBINAR_DATE_TIME_FORMAT)
```

Comment:
This is being used at more then one place, Can we create a helper?

Reply:
yes created a helper and used it on both edx platform and theme side.